CONTRIBUTING.md

```diff
@@ -2,25 +2,44 @@
 
 **This document applies during 2024 and may change at a future date.**
 
-### Introduction
+## Introduction
 
-The current status of the ASVS project is as follows:
+### What is [OWASP](https://owasp.org/)?
+
+The Open Worldwide Application Security Project (OWASP) is a nonprofit organization that works to improve the security of software. It has many programs to work towards this goal. One of those programs is the ASVS.
+
+### What is the [ASVS](https://owasp.org/www-project-application-security-verification-standard/)?
+
+The OWASP Application Security Verification Standard (ASVS) Project provides a basis for testing web application technical security controls and also provides developers with a list of requirements for secure development.
+
+The primary aim of the OWASP Application Security Verification Standard (ASVS) Project is to normalize the range in the coverage and level of rigor available in the market when it comes to performing Web application security verification using a commercially-workable open standard.
+
+### What is the Current Status of ASVS development?
 
 The ASVS project is planning to release a 5.0 version during 2024 which will be a significant modification, similar to the scale of changes which happened during 4.0.
- 
-We will try, on a best efforts basis, to address issues and push changes to the "bleeding edge" master branch within the 5.0 folder **without altering current numbering** and using tagging ([see below](CONTRIBUTING.md#use-tags-to-describe-the-change)) to make changes clearer. This is in order to make it easier to use this branch on an ongoing basis. However, we will no longer be accepting changes to the 4.0 folder which is now fixed at the 4.0.3 release.
 
-Please also focus attention on the requirements themselves and **not** on the surrounding text or on the introductory chapters (files 0x01-0x04). This because the text might have become outdated where requirements have changed and also because our goal is to significantly cut down this text anyway in 5.0.
+We will try, on a best efforts basis, to address issues and push changes to the "bleeding edge" master branch within the 5.0 folder **without altering current numbering** and using tagging ([see below](CONTRIBUTING.md#use-tags-to-describe-the-change)) to make changes clearer. This is in order to make it easier to use this branch on an ongoing basis. 
+
+We will no longer be accepting changes to the 4.0 folder which is now fixed at the 4.0.3 release.
 
-### Where do I start?
+Please also focus attention on the requirements themselves and **not** on the surrounding text or on the introductory chapters (files 0x01-0x04). This is because the text might have become outdated where requirements have changed and also because our goal is to significantly cut down this text in version 5.0.
```

Comment:
cleanup: "Please focus only on the requirements and not on the surrounding text or introductory chapters (files 0x01-0x04). The text might be outdated due to changes in requirements, and we aim to reduce it significantly in version 5.0."

Reply:
Done